mos_tests/neutron/python_tests/test_dvr.py

```diff
@@ -34,20 +35,27 @@ def variables(self, init):
         self.security_group = self.os_conn.create_sec_group_for_ssh()
         self.instance_keypair = self.os_conn.create_key(key_name='instancekey')
 
-    def test_vm_connectivity_wo_floating(self, variables):
-        """Check North-South connectivity (without floating IP)
+    @pytest.mark.parametrize('floating_ip', (True, False),
+                             ids=('with floating', 'without floating'))
+    @pytest.mark.parametrize('dvr_router', (True, False),
+                             ids=('distributed router', 'centralized_router'))
+    def test_north_south_connectivity(self, variables, floating_ip,
+                                         dvr_router):
+        """Check North-South connectivity
 
         Scenario:
             1. Create net01, subnet net01__subnet for it
             2. Create router01 with external network and
-                router type Distributed
+                router type `dvr_router`
             3. Add interfaces to the router01 with net01__subnet
             4. Boot vm_1 in the net01
-            5. Go to the vm_1
-            6. Ping 8.8.8.8
+            5. Add floationg ip if case of `floating_ip` arg is True
+            6. Go to the vm_1
+            7. Ping 8.8.8.8
         """
         net, subnet = self.create_internal_network_with_subnet(1)
-        router = self.os_conn.create_router(name='router01', distributed=True)
```

Comment:
But fn mos_tests.environment.os_actions.OpenStackActions#create_router already changed, you can change and 
mos_tests.neutron.python_tests.base.TestBase#create_router_between_nets like this def create_router_between_nets(self, ext_net, subnet, suffix=1, distributed=False):